examples/duel/www/index.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
```

Comment:
That's a generated file. Would require a change to DUEL. Funny I'd forgotten that it does caps. I may make that change in DUEL but not this commit.